lldb/test/API/functionalities/asan/TestMemoryHistory.py

```diff
@@ -26,6 +33,68 @@ def setUp(self):
         self.line_free = line_number("main.c", "// free line")
         self.line_breakpoint = line_number("main.c", "// break line")
 
+    # Test line numbers: rdar://126237493
+    def libsanitizer_tests(self):
+        target = self.createTestTarget()
+
+        self.runCmd(
```

Comment:
Another instance of where we will need to remove these in the future when they're no longer needed.